objection_engine/anim.py

```diff
@@ -298,21 +314,30 @@ def do_video(config: List[Dict], output_filename, resolution_scale):
     return sound_effects
 
 def do_audio(sound_effects: List[Dict], output_filename):
+    """
+    Renders the sound for the video given the data outputted by do_video().
+    """
+    # Track containing sound effects
     audio_se = AudioSegment.empty()
-    bip = AudioSegment.from_wav(
-        "assets/sfx general/sfx-blipmale.wav"
-    ) + AudioSegment.silent(duration=50)
+
+    # Character speech sound effect
+    bip = AudioSegment.from_wav("assets/sfx general/sfx-blipmale.wav") + AudioSegment.silent(duration=50)
+    long_bip = bip * 100
+    long_bip -= 10
+
+    # "Next dialogue" sound effect
     blink = AudioSegment.from_wav("assets/sfx general/sfx-blink.wav")
     blink -= 10
+
+    # "Objection!" sound effect for characters without voiced objections(?)
```

Comment:
This audio is meant as a short "shocked" sound effect. The "default" objection for those who doesn't have an specific voiceover is `default_objection` as specified on line 338